frontend/src/component/feature/FeatureView/FeatureOverview/ReleasePlan/ChangeRequest/ReleasePlanAddChangeRequestDialog.tsx

```diff
@@ -45,7 +62,7 @@ export const ReleasePlanAddChangeRequestDialog = ({
     return (
         <Dialogue
             title='Request changes'
-            open={Boolean(releaseTemplate)}
+            open={Boolean(releaseTemplate) || Boolean(releasePlan)}
```

Comment:
Not a huge fan of this ambiguity. What is this dialog doing that it shows up if one of these is truthy? I also prefer to handle the open state explicitly instead of relying on other state variables, but that's mostly a personal preference.

Reply:
This is also gone, very strange